diff --git a/lib/cmake/Qt5Core/Qt5CoreConfig.cmake b/lib/cmake/Qt5Core/Qt5CoreConfig.cmake
index 5bad1af..25bf3e3 100644
--- a/lib/cmake/Qt5Core/Qt5CoreConfig.cmake
+++ b/lib/cmake/Qt5Core/Qt5CoreConfig.cmake
@@ -264,7 +264,7 @@ if (NOT TARGET Qt5::Core)
         return()
     endif()
 
-    set(_Qt5Core_LIB_DEPENDENCIES "")
+    set(_Qt5Core_LIB_DEPENDENCIES "${_qt5Core_install_prefix}/lib/qtpcre2.lib;netapi32.lib;version.lib")
 
 
     if(NOT Qt5_EXCLUDE_STATIC_DEPENDENCIES)
diff --git a/lib/cmake/Qt5Widgets/Qt5WidgetsConfig.cmake b/lib/cmake/Qt5Widgets/Qt5WidgetsConfig.cmake
index d9966ad..308e4cb 100644
--- a/lib/cmake/Qt5Widgets/Qt5WidgetsConfig.cmake
+++ b/lib/cmake/Qt5Widgets/Qt5WidgetsConfig.cmake
@@ -264,7 +264,7 @@ if (NOT TARGET Qt5::Widgets)
         return()
     endif()
 
-    set(_Qt5Widgets_LIB_DEPENDENCIES "Qt5::Gui;Qt5::Core")
+    set(_Qt5Widgets_LIB_DEPENDENCIES "Qt5::Gui;Qt5::Core;dwmapi.lib;uxtheme.lib")
 
 
     if(NOT Qt5_EXCLUDE_STATIC_DEPENDENCIES)
